pages/404.html

```diff
@@ -4,18 +4,48 @@
 ---
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
 	<meta charset="UTF-8" />
 	<title>Can I WebView… 404</title>
 	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 	<style>
-		*{margin:0;padding:0;box-sizing:border-box}body{display:flex;align-items:center;justify-content:center;padding:0 1em;min-width:17.5rem;min-height:100vh;font:1em/1.5 -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;color:#b1b1b3;background:#1b1b1d;scroll-behavior:smooth}h1{color:#fff}img{display:block;margin:1rem auto;filter: invert(1)}
+		* {
+			margin: 0;
+			padding: 0;
+			box-sizing: border-box
+		}
+
+		body {
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			padding: 0 1em;
+			min-width: 17.5rem;
+			min-height: 100vh;
+			font: 1em/1.5 -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
+			color: #b1b1b3;
+			background: #1b1b1d;
+			scroll-behavior: smooth
+		}
+
+		h1 {
+			color: #fff
+		}
+
+		img {
+			display: block;
+			margin: 1rem auto;
+			filter: invert(1)
```

Comment:
It looks like logo.svg isn't used for anything so we can update the style of the logo directly and remove the need for a filter. Should be a little faster.

We can just open logo.svg and add something like this:

```
<style>
        path {
            fill: white;
        }
    </style>
```

Also would be worth renaming that svg to something like not found.